plugins/nid-team/commands/sync-pr-dashboard.md

```
@@ -0,0 +1,97 @@
+---
+description: Sync the NI&D GitHub Project board — populate PR authors, sync reviewers to assignees, add shared repo PRs, classify areas
+argument-hint: ""
+---
+
+## Name
+
+nid-team:sync-pr-dashboard
+
+## Synopsis
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)